src/screens/WelcomeScreen.js

```diff
@@ -1,78 +1,265 @@
-    } catch (err) {
-      console.log('Error to fetch data:', err);
+  renderLottie(){
+    if (this.state.emojiNum == 0) {
+      return <LottieView style={{flex:1, width: 10}} source={require('../lotties/flushed.json')} autoPlay loop/>
+    }
+    if (this.state.emojiNum == 1) {
+      return <LottieView style={{flex:1, width: 10}} source={require('../lotties/shocked.json')} autoPlay loop/>
+    }
+    if (this.state.emojiNum == 2) {
+      return <LottieView style={{flex:1, width: 10}} source={require('../lotties/silly.json')} autoPlay loop/>
+    }
+    if (this.state.emojiNum == 3) {
+      return <LottieView style={{flex:1, width: 10}} source={require('../lotties/sleeping.json')} autoPlay loop/>
+    }
+    if (this.state.emojiNum == 4) {
+      return <LottieView style={{flex:1, width: 10}} source={require('../lotties/vomiting.json')} autoPlay loop/>
     }
+  }
+
+
+  start = async () => {
+    const {increasePageNum} = this.props;
+    this.setState({
+      loading: true,
+      settingsDisabled: true,
+    });
+
+    const URL =
+      'https://opentdb.com/api.php?amount=15&encode=url3986&category=' +
+      this.state.category +
+      '&difficulty=' +
+      this.state.difficulty +
+      '&type=multiple';
+
+    const questionApiCall = await fetch(URL)
+
+    const data = await questionApiCall.json();
+    this.props.updateQuestionData(data);
+
+    console.log(this.props.questionReducer.QUESTIONS.results[
+      this.props.questionReducer.questNum
+    ].correct_answer)
 
     const answers = [
       ...this.props.questionReducer.QUESTIONS.results[
-        this.props.questionReducer.QuestNum
+        this.props.questionReducer.questNum
       ].incorrect_answers,
       this.props.questionReducer.QUESTIONS.results[
-        this.props.questionReducer.QuestNum
+        this.props.questionReducer.questNum
       ].correct_answer,
     ];
 
+    
+
     let activeQuestion = [];
 
     activeQuestion[
       QUESTION_INDEX
     ] = this.props.questionReducer.QUESTIONS.results[
-      this.props.questionReducer.QuestNum
+      this.props.questionReducer.questNum
     ].question;
 
     randomizedAnswers = shuffle(answers);
     activeQuestion[ANSWER_INDEX] = randomizedAnswers;
 
     this.props.updateActiveQuestion(activeQuestion);
+
+    this.setState({
+      loading: false,
+      settingsDisabled: false,
+    });
+
+    increasePageNum(this.props.pageReducer.pageNum);
+  };
+
+  displayModal(show) {
+    this.setState({isVisible: show});
   }
+
+  button() {
+    const {buttonStyle} = styles;
+    if (this.state.loading == true) {
+      return (
+        <View>
+          <ActivityIndicator size="large"></ActivityIndicator>
+        </View>
+      );
+    }
+    if (this.state.loading == false) {
+      return (
+        <View>
+          <TouchableOpacity onPress={this.start}>
+            <Text style={buttonStyle}>GET STARTED</Text>
+          </TouchableOpacity>
+        </View>
+      );
```

Comment:
if(this.state.loading) yazmak ===true ile aynı

Reply:
İki if koymaya gerek yok if else yapmak lazım. 

{this.state.loading? <Activityindicator> : <diğer view>}

Reply:
got it